server/polar/account/endpoints.py

```diff
@@ -18,30 +19,36 @@ async def create_account(
     auth: Auth = Depends(Auth.user_with_org_access),
     session: AsyncSession = Depends(get_db_session),
 ) -> AccountRead:
-    created = await account_service.create_account(
-        session, auth.organization.id, auth.user.id, account
-    )
-    if not created:
-        raise HTTPException(status_code=400, detail="Error while creating account")
+    try:
+        created = await account_service.create_account(
+            session, auth.organization.id, auth.user.id, account
+        )
+    except AccountServiceError as e:
+        raise HTTPException(status_code=400, detail=e.message)
+
     return AccountRead.from_orm(created)
 
 
 @router.get(
-    "/{platform}/{org_name}/accounts/{stripe_id}/onboarding_link",
+    "/{platform}/{org_name}/accounts/{account_id}/onboarding_link",
     response_model=AccountLink,
 )
 async def onboarding_link(
     platform: Platforms,
     org_name: str,
-    stripe_id: str,
+    account_id: UUID4,
```

Comment:
I can't believe I didn't know about this built-in pydantic type. TIL. So nice. I think we have quite a few places where we're using `str` today. Will update & enhance as we progress throughout the codebase now knowing this.

Reply:
Yes, very convenient! It automatically converts the parameter to a proper UUID object (or throws a validation error if not possible).